auth-web/tests/unit/components/Product.spec.ts

```diff
@@ -159,19 +159,18 @@ describe('Product.vue', () => {
     expect(wrapper.vm.hasDecisionNotBeenMade).toBeFalsy()
   })
 
-  it('premium product should be disabled in basic account settings', async () => {
+  it('premium product should be visiable to basic account settings', async () => {
```

Comment:
Your changes should not be effecting the Product component. Premium only things in here should still be premium only and have the minus checkboxes visible.